Tools/ardupilotwaf/toolchain.py

```diff
@@ -126,13 +126,17 @@ def find_toolchain_program(cfg, filename, **kw):
     return cfg.find_program(filename, **kw)
 
 def configure(cfg):
+    _filter_supported_c_compilers('gcc', 'clang')
+    _filter_supported_cxx_compilers('g++', 'clang++')
+    
     if cfg.env.TOOLCHAIN == 'native':
         cfg.load('compiler_cxx compiler_c')
+        if cfg.env.COMPILER_CC == 'clang' or cfg.env.COMPILER_CXX == 'clang++':
+            if cfg.env.DEST_OS == 'darwin':
+                Logs.pprint('RED', "Warning! native clang build could be unstable on macOS, Please use GCC/G++. \
```

Comment:
Any reason not to use `Logs.warn`?

Reply:
do you know of alternative?

Reply:
`Logs.warn("message")` is the alternative 🙂 By the way, I'm not sure that `\n` is handled correctly in Windows, but we can fix that later if it isn't.